sdk/src/test/scala/com/horizen/api/http/SidechainApiRouteTest.scala

```diff
@@ -102,6 +102,9 @@ abstract class SidechainApiRouteTest extends WordSpec with Matchers with Scalate
         case GetDataFromCurrentSidechainNodeView(f) =>
           if (sidechainApiMockConfiguration.getShould_nodeViewHolder_GetDataFromCurrentSidechainNodeView_reply())
             sender ! f(utilMocks.getSidechainNodeView(sidechainApiMockConfiguration))
+        case ApplyFunctionOnNodeView(f) =>
```

Comment:
Add a test case for `ApplyBiFunctionOnNodeView` as well.

Reply:
will be done later